### PR TITLE
Add examples using cert/key pair

### PR DESCRIPTION
**Pull request by zbauman3**
Add examples using cert/key pair to address https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/issues/98

### CODE REVIEW

File: examples/native_networking/minimqtt_adafruitio_native_networking.py [hunk 1]
```diff
@@ -59,6 +60,14 @@ def message(client, topic, message):
 
 # Create a socket pool
 pool = socketpool.SocketPool(wifi.radio)
+ssl_context = ssl.create_default_context()
+
+# If you need to use certificate/key pair authentication (e.g. X.509), you can load them in the
+# ssl context by uncommenting the line below
```
Comment (brentru) [inline examples/native_networking/minimqtt_adafruitio_native_networking.py:66]: @zbauman3 Might also want to specify in your comment that these two files should be located within the secrets file's `device_cert_path` and `device_key_path`

Comment (zbauman3) [reply examples/native_networking/minimqtt_adafruitio_native_networking.py:66]: Will do! AFK right now, I'll update and tag you tomorrow 👍

Comment (zbauman3) [reply examples/native_networking/minimqtt_adafruitio_native_networking.py]: @brentru I added some more detail to the comments. Thanks for the suggestion!

**Review by brentru — APPROVED** [review]

**Comment by zbauman3** [comment]
@brentru no problem! I've been loving using it, y'all have done an amazing job!

I think this should mean you can close #98 👌

**Pull request merged**